Modelica/Electrical/Analog/Examples/Lines/LightningLosslessTransmissionLine.mo

```diff
@@ -36,7 +36,7 @@ model LightningLosslessTransmissionLine
     annotation (Placement(transformation(extent={{-10,-30},{10,-10}})));
   Sources.LightningImpulseCurrent
                         lightningImpulseCurrent(
-    approximation=Modelica.Electrical.Analog.Types.ImpulseApproximation.DoubleExp,
+    approximation=Modelica.Electrical.Analog.Types.ImpulseApproximation.Heidler,
     I=10e3,
```

Comment:
The original examples from @max-privato did use DoubleExp. But probably not an issue?

Reply:
I'd prefer to set Heidler as default, since this function is even used in the respective standard.

Reply:
I see. Fine with me.